src/lib/render-image.js

```diff
@@ -42,24 +40,35 @@ function renderImageFromHtml(html, outputPath) {
       }
 
       Jimp.read(tempFile)
-        .then(image => image.autocrop().write(outputPath))
+        .then(cropAndWriteFile.bind(null, outputPath))
```

Comment:
you can also just return a function that invokes cropAndWriteFile, like: 
`.then(() => cropAndWriteFile(outputPath))`
which is a tiny bit cleaner.